bin/pycbc_live

```diff
@@ -1160,11 +1168,11 @@ with ctx:
                 # Pick the best coinc in this chunk
                 best_coinc = Coincer.pick_best_coinc(coinc_results)
 
-                evnt.check_coincs(list(results.keys()), best_coinc, psds)
+                evnt.check_coincs(list(results.keys()), best_coinc, psds, skymap_only_ifos=skymap_only_ifos)
 
             # Check for singles
             if args.enable_single_detector_background:
-                evnt.check_singles(results, psds)
+                evnt.check_singles(results, psds, skymap_only_ifos=skymap_only_ifos)
```

Comment:
I think you can simplify the code if you set `skymap_only_ifos` to be an attribute of the `LiveEventManager` class. Then you do not need to carry around all these arguments.

Reply:
This is what I had initially thought, but dropped the idea seeeing that ```ifos``` is not an attribute of the ```LiveEventManager```. Is there  a reason. Now I have added ```skymap_only_ifos``` as an attribute which seems to have simplified things. But the [test](https://github.com/gwastro/pycbc/actions/runs/4884286183/jobs/8716818008#step:4:47) failed which I cannot understand atm. Where should I look at to debug.

Reply:
The test failure seems to be a condor installation issue - I will click to restart failed tests in case this was a transient thing, but we may need a fix elsewhere if it isnt